docs/advanced/scf/spin.md

```diff
@@ -1,5 +1,41 @@
+
+Magnetic moment Settings in [STRU files](docs/advanced/input_files/stru.md) are not ignored until **"nspin 2"** is set in INPUT file
+
+When **"nspin 2"** is set, the screen output file will contain magnetic moment information. e.g.
+![image](https://user-images.githubusercontent.com/49852742/192320064-d857858a-de83-43f7-ab07-9923d27eb646.png)
+where "TMAG" refers to total magnetization and "AMAG" refers to average magnetization.
+For more detailed orbital magnetic moment information, please use [Mulliken charge analysis](docs/advanced/elec_properties/Mulliken.md).
+
+## Noncollinear Spin Polarized Calculations
+The spin non-collinear polarization calculation corresponds to setting **"noncolin 1"**, in which case the coupling between spin up and spin down will be taken into account. 
+In this case, nspin is automatically set to 4, which is usually not required to be specified manually.
+The weight of each band will not change, but the number of occupied states will be double. 
+If the nbands parameter is set manually, it is generally set to twice what it would be when nspin<4.
+
+In general, non-collinear magnetic moment settings are often used in calculations considering [SOC effects](docs/advanced/scf/soc.md). When **"lspinorb 1"** in INPUT file, "nspin" is also automatically set to 4. 
+Note: different settings for "noncolin" and "lspinorb" correspond to different calculations:
+ - noncolin=0 lspinorb=0 nspin<4 : 
+Non-collinear magnetic moments and SOC effects are not considered.
+ - noncolin=0 lspinorb=0 nspin=4 : 
+Actualy same as the above setting, but the calculation will be larger. So the setting is not recommended.
+ - noncolin=1 lspinorb=0 : 
+Non-collinear magnetic moments are considered but SOC effects are not considered
+ - noncolin=0 lspinorb=1 : 
+The SOC effect is considered but the magnetic moment is limited to the Z direction
+ - noncolin=1 lspinorb=1 : 
+The SOC effect and non-collinear magnetic moment are both calculated.
+
+## For the continuation job
+- Continuation job for "nspin 1" need file "SPIN1_CHG" which is generated by setting "out_chg=1" in task before. By setting "init_chg file" in new job's INPUT file, charge density will start from file but not atomic. 
+- Continuation job for "nspin 2" need files "SPIN1_CHG" and "SPIN2_CHG" which are generated by "out_chg 1" with "nspin 2", and refer to spin-up and spin-down charge densities respectively. It should be note that reading "SPIN1_CHG" only for the continuation target magnetic moment job is not supported now.
+- Continuation job for "nspin 4" need files "SPIN$s_CHG", where $s in {1,2,3,4}, which are generated by "out_chg 1" with any variable setting leading to 'nspin'=4, and refer to charge densities in Pauli spin matrixes. It should be note that reading charge density files printing by 'nspin'=2 case is supported, which means only $$\sigma_{tot}$$and $$\sigma_{z}$$are read.
```

Comment:
Please avoid using "$" in markdown text. This will break the rich-text rendering. See https://abacus-rtd--1320.org.readthedocs.build/en/1320/advanced/scf/spin.html#for-the-continuation-job for preview.